Test/FMI1/fmi1_xml_parsing_test.c

```diff
@@ -11,14 +11,14 @@ static const int NO_LOG_EXPECTED_MSG = 0;
 static const int DO_LOG_EXPECTED_MSG = 1;
 
 static int did_log_expected_msg;
-static char *expected_message = "Invalid structured ScalarVariable name";
-static char *name_check_test_directory;
+static char *g_name_check_test_directory;
+static char *g_expected_message = "Invalid structured ScalarVariable name";
```

Comment:
Not a warnings fix, but added this as well. I think we should use `s_` for static variables instead of `g_` but changed it like this for consistency for now.

Reply:
Well they are not truly global, they are still local to the file/compilation unit. I meant we'd then distinguish between:

```
char* g_str = ...;
static char* s_str = ...;
```

I mean, FMIL has no global variable, only static ones, so would prefer to call them for what they are.